### PR TITLE
Support generate config

### PR DESCRIPTION
**Pull request by seanmcoleman**
@aflanagan I'm hitting a 500 error at https://cronitor.io/api/monitors.yaml, which I think is the right URL to generate the full config. I might be missing something obvious, but this code should match the Python implementation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [Cronitor](https://cronitor.io/) provides dead simple monitoring for cron jobs, daemons, data pipelines, queue workers, and anything else that can send or receive an HTTP request. The Cronitor Ruby library provides convenient access to the Cronitor API from applications written in Ruby.
```
Comment (aflanagan) [inline README.md:3]: @seanmcoleman could you tuck the github tests passing badge in here?

**Review by aflanagan — APPROVED** [review]

File: lib/Monitor.php [hunk 2]
```diff
@@ -60,6 +60,18 @@ public static function put($apiKey, $apiVersion, $params = [])
         }
     }
 
+    public static function getYaml($apiKey, $apiVersion)
+    {
+        $client = self::getMonitorHttpClient($apiKey, $apiVersion);
+        $response = $client->get('.yaml', ['timeout' => 10]);
```
Comment (aflanagan) [inline lib/Monitor.php]: we might want to allow a slightly longer timeout -- users with a large number of monitors (>100) might not actually get a file generated in 10s currently 😬

Comment (seanmcoleman) [reply lib/Monitor.php]: Good point! Even a small config was taking 2-3 seconds to fully generate and download. I set to 25 seconds. I think the Python library is set at 10 seconds FYI.

Comment (aflanagan) [reply lib/Monitor.php]: thanks for calling that out! Will update it.

**Pull request merged**